localization/pose_estimator/ndt_scan_matcher/ndt_scan_matcher/CMakeLists.txt

```diff
@@ -1,57 +1,20 @@
-cmake_minimum_required(VERSION 3.0.2)
+cmake_minimum_required(VERSION 3.5)
 project(ndt_scan_matcher)
 
-add_compile_options(-std=c++14)
+if(NOT CMAKE_CXX_STANDARD)
+  set(CMAKE_CXX_STANDARD 14)
+endif()
 
-find_package(catkin REQUIRED COMPONENTS
-  roscpp
-  tf2
-  tf2_ros
-  tf2_sensor_msgs
-  tf2_geometry_msgs
-  std_msgs
-  geometry_msgs
-  sensor_msgs
-  nav_msgs
-  visualization_msgs
-  diagnostic_msgs
-  pcl_conversions
-  pcl_ros
-  dynamic_reconfigure
-  ndt_pcl_modified
-  ndt_omp
-  ndt
-  autoware_localization_srvs
-)
-
-catkin_package(
-  INCLUDE_DIRS include
-)
+if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+  add_compile_options(-Wall -Wextra -Wpedantic)
+endif()
 
-include_directories(include ${catkin_INCLUDE_DIRS})
+find_package(ament_cmake_auto REQUIRED)
+ament_auto_find_build_dependencies()
 
-add_executable(ndt_scan_matcher
+ament_auto_add_executable(ndt_scan_matcher
   src/ndt_scan_matcher_node.cpp
   src/ndt_scan_matcher_core.cpp
 )
 
-add_dependencies(ndt_scan_matcher
-  ${${PROJECT_NAME}_EXPORTED_TARGETS}
-  ${catkin_EXPORTED_TARGETS}
-)
-
-target_link_libraries(ndt_scan_matcher ${catkin_LIBRARIES})
-
-install(
-  TARGETS
-    ndt_scan_matcher
-  ARCHIVE DESTINATION ${CATKIN_PACKAGE_LIB_DESTINATION}
-  LIBRARY DESTINATION ${CATKIN_PACKAGE_LIB_DESTINATION}
-  RUNTIME DESTINATION ${CATKIN_PACKAGE_BIN_DESTINATION}
-)
-
-install(
-  DIRECTORY
-    launch
-  DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION}
-)
+ament_auto_package()
```

Comment:
I think you may need to add `INSTALL_TO_SHARE launch` here? Else I couldn't find the launch files post sourcing the setup.bash

Reply:
Thanks, you're right!